File: CS/DevExpress.Xpf.DnD/DragAndDrop/IDragAndDropHandler.cs
using System;
using System.Windows;
using System.Windows.Input;

namespace DX.Xpf.DnD {
    // Provides app level feedback to the Drag and Drop manager
    public interface IDragAndDropHandler {
        bool CanStartDrag(UIElement source, MouseButtonEventArgs e, ref object dragData);
        FrameworkElement CreateDragThumb(UIElement source, UIElement target, object dragData);
    }
}

File: VB/DevExpress.Xpf.DnD/DragAndDrop/IDragAndDropHandler.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows
Imports System.Windows.Input

Namespace DX.Xpf.DnD
	' Provides app level feedback to the Drag and Drop manager
	Public Interface IDragAndDropHandler
		Function CanStartDrag(ByVal source As UIElement, ByVal e As MouseButtonEventArgs, ByRef dragData As Object) As Boolean
		Function CreateDragThumb(ByVal source As UIElement, ByVal target As UIElement, ByVal dragData As Object) As FrameworkElement
	End Interface
End Namespace